src/SmartNumber.php

```diff
@@ -212,7 +247,7 @@ public function __toString(): string
 	 */
 	public function getString(): string
 	{
-		return $this->getLatex(true);
+		return $this->__toString();
```

Comment:
It looks too magic. Can you call regular method instead of magic?

Reply:
Ok, done `return (string) $this;`